protected/views/gp/index.php

```diff
@@ -43,27 +46,27 @@
           <tr id="r<?php echo $gp->id; ?>" class="clickable">
             <td><?php echo CHtml::encode($gp->getCorrespondenceName()); ?></td>
             <td><?php echo CHtml::encode($gp->contact->primary_phone); ?></td>
-            <td><?php echo Chtml::encode(isset($gp->contact->label)?$gp->contact->label->name:'')?></td>
+            <td><?php echo Chtml::encode(isset($gp->contact->label) ? $gp->contact->label->name : '') ?></td>
```

Comment:
*CHtml

I would think capitalisation of class names would be important in PHP, but if this works then clearly it isn't.

Reply:
Fixed